From fb54ab619202903033d6b2763ac0bdf10870fd86 Mon Sep 17 00:00:00 2001
From: Gatien Chevallier <gatien.chevallier@foss.st.com>
Date: Fri, 26 Jan 2024 13:19:54 +0100
Subject: [PATCH 0568/1695] dts: stm32: add backup registers configuration for
 stm32mp135f-dk

Add a backup register configuration for this board.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 76c63fa7e6bc5fac7d6041618a55dfba571e1693 dts: stm32: add backup registers configuration for stm32mp135f-dk]
Signed-off-by: Gatien Chevallier <gatien.chevallier@foss.st.com>
Change-Id: Ibd2f82e1ebab3112c06483f2ec813ab601d83832
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/354736
Tested-by: Maxime MERE <maxime.mere@foss.st.com>
Reviewed-by: Gatien CHEVALLIER <gatien.chevallier@st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Yann GAUTIER <yann.gautier@foss.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
Reviewed-by: Maxime MERE <maxime.mere@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
---
 core/arch/arm/dts/stm32mp135f-dk.dts | 1 +
 1 file changed, 1 insertion(+)

diff --git a/core/arch/arm/dts/stm32mp135f-dk.dts b/core/arch/arm/dts/stm32mp135f-dk.dts
index 0586eceac..21433fb0f 100644
--- a/core/arch/arm/dts/stm32mp135f-dk.dts
+++ b/core/arch/arm/dts/stm32mp135f-dk.dts
@@ -568,6 +568,7 @@
 
 &tamp {
 	status = "okay";
+	st,backup-zones = <10 5 17>;
 	st,tamp-passive-nb-sample = <4>;
 	st,tamp-passive-sample-clk-div = <16384>;
 
-- 
2.47.2

